drivers/modem/modem_cellular.c

```diff
@@ -57,7 +57,7 @@ enum modem_cellular_event {
 	MODEM_CELLULAR_EVENT_BUS_CLOSED,
 };
 
-static bool rtc_synced = false;
+bool modem_cellular_rtc_synced = false;
```

Comment:
false is default, therefore the ci fails, no need to set false on global (or 0).